CryptoPrices/Sources/MyCoin/Sources/MyCoin/ChartSectionView/PriceLineChartView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  PriceLineChartView.swift
+//  MyCoin
+//
+//  Created by Minh Pham on 04/01/2023.
+//
+// swiftlint:disable file_types_order
+
+import Charts
+import Styleguide
+import SwiftUI
+
+struct PriceLineChartView: UIViewRepresentable {
+
+    let entries: [ChartDataEntry]
+
+    let lineChart = LineChartView()
```

Comment:
I think the break line is not needed, is it? :D.
```suggestion
    let entries: [ChartDataEntry]
    let lineChart = LineChartView()
```